packages/flutter/lib/src/services/scribe.dart

```diff
@@ -0,0 +1,129 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'message_codec.dart';
+import 'system_channels.dart';
+
+/// An interface into Android's stylus handwriting text input.
+///
+/// Allows handwriting directly on top of a text input using a stylus.
+///
+/// See also:
+///
+///  * [EditableText.stylusHandwritingEnabled], which controls whether Flutter's
+///    built-in text fields support handwriting input.
+///  * [SystemChannels.scribe], which is the [MethodChannel] used by this
+///    class, and which has a list of the methods that this class handles.
+final class Scribe {
+  static const MethodChannel _channel = SystemChannels.scribe;
+
+  /// A convenience method to check if the device currently supports Scribe
+  /// stylus handwriting input.
+  ///
+  /// Call this each time before calling [startStylusHandwriting] to make sure
+  /// it's available.
+  ///
+  /// The return type is nullable due to the way
+  /// [MethodChannel.invokeMethod](https://main-api.flutter.dev/flutter/services/MethodChannel/invokeMethod.html)
+  /// works, but a successful call will never resolve to `null`.
```

Comment:
Would it be possible to do `.then<bool>((bool? value) => value ?? false)`

or `.then<bool>((bool? value) => value ?? throw SomeError)` then?

Reply:
Good call, I should do this checking and not the user.